.env

```diff
@@ -1 +1 @@
-PYTHONPATH=${PYTHONPATH}
+PYTHONPATH=${PYTHONPATH}:${PWD}
```

Comment:
Had to undo a change made in #845 as it was causing my tests to fail, unsure of if there was a reason that this was changed and if I should look for an alternate solution